packages/react-devtools-shared/src/backend/console.js

```diff
@@ -49,19 +52,16 @@ export function dangerous_setTargetConsoleForTesting(
 // These internals will be used if the console is patched.
 // Injecting them separately allows the console to easily be patched or un-patched later (at runtime).
 export function registerRenderer(renderer: ReactRenderer): void {
-  const {getCurrentFiber, findFiberByHostInstance, version} = renderer;
+  const {getCurrentFiber, findFiberByHostInstance} = renderer;
 
   // Ignore React v15 and older because they don't expose a component stack anyway.
   if (typeof findFiberByHostInstance !== 'function') {
     return;
   }
 
   if (typeof getCurrentFiber === 'function') {
-    const {getDisplayNameForFiber} = getInternalReactConstants(version);
-
     injectedRenderers.set(renderer, {
       getCurrentFiber,
-      getDisplayNameForFiber,
```

Comment:
Hm... actually now that I think about it, won't this importing rather than injecting change cause problems, @sebmarkbage? b'c of multiple React versions.

Reply:
Just forking seems like a better option then. It's not much code and that allows us to cleanly decouple it without compromising on PROD size.

Reply:
Currently it's the same format for PROD and DEV but one idea would be to do a more advanced but different technique in DEV that rerenders the tree.

One thing we might want to consider is dropping the DEV stacks from the DEV build and only make it a separate package that you install that overrides console.error e.g. for jest environments like a light weight devtools. Then the DevTools injected form would be the only implementation of the DEV stacks which would be a different implementation than the prod one passed to error boundaries.

Reply:
Forking (to support older versions too) will be more complicated but maybe it's the right thing to do. I'll take a pass at it today.